xds/src/main/java/io/grpc/xds/LookasideChannelLb.java

```diff
@@ -76,11 +51,9 @@ public void onReportResponse(long reportIntervalNano) {
         };
     this.lrsClient = lrsClient;
 
-    AdsStreamCallback adsCallback = new AdsStreamCallbackImpl(
+    XdsClient.EndpointWatcher endpointWatcher = new EndpointWatcherImpl(
```

Comment:
`XdsClient.EndpointWatcher` -> `EndpointWatcher`

since `XdsClient.EndpointWatcher` is imported

Reply:
Done.